src/seedu/addressbook/data/person/ReadOnlyPerson.java

```diff
@@ -83,4 +83,16 @@ default String getAsTextHidePrivate() {
         }
         return builder.toString();
     }
+    
+    /**
+     * Returns a concatenated version of the printable strings of each object.
+     */
+    default String getPrintableString(Printable... printables){
+        String printableString = "";
+        for(Printable printable : printables){
+            printableString += printable.getPrintableString();
+            printableString += ",";
+        }
```

Comment:
Doesn't this result in an extra `,` at the end? You could have used `StringJoiner` or `String.join` instead.